extensions/ffi/lpffiwrappers.pas

```diff
@@ -822,7 +827,7 @@ function LapeFFIPointerParam(const Param: TLapeParameter; ABI: TFFIABI): Boolean
 
   {$IF DECLARED(FFI_WIN64)}
       or ((ABI = FFI_WIN64) and (Param.VarType.BaseType = ltRecord) and (not (UInt8(Param.VarType.Size) in PowerTwoRegs)))
-      or ((ABI = FFI_WIN64) and (Param.VarType.BaseType = ltRecord) and (Param.ParType in Lape_ConstParams) and (Param.VarType.Size = SizeOf(UInt8)))
+      or ((ABI = FFI_WIN64) and (Param.VarType.BaseType = ltRecord) and (Param.ParType in Lape_ConstParams) and (Param.VarType.Size in PowerTwoRegs))
```

Comment:
You can add `or (Param.ParType in Lape_ConstParams)` to the previous statement, as they almost check for the same condition.